scripts/install_requirements.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+from ConfigParser import ConfigParser
+
+import argparse
+import os
+import subprocess
+import sys
+
+
+def installReqs(file, pip):
+    if not os.path.exists(file):
+        return
+
+    print('\033[32m*** Installing: %s\033[0m' % file)
+    if subprocess.call([pip, 'install', '-U', '-r', file]) != 0:
```

Comment:
I think this would be safer (at least as a default) given multiple python installs:
```python
import pip
pip.main(['install', '-U', '-r', file])
```

I can't think of a reason you would use a pip not in the current `PYTHONPATH`.